applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -145,7 +145,7 @@ def _check_outputs(self,mp,A,dim):
         E = mp.GetProperties()[1].GetValue(KratosMultiphysics.YOUNG_MODULUS)
         NU =mp.GetProperties()[1].GetValue(KratosMultiphysics.POISSON_RATIO)
         
-        #given the matrix A, the analytic deformation graident is F+I
+        #given the matrix A, the analytic deformation gradient is F+I
         F = A
         for i in range(3):
```

Comment:
@loumalouomega , please try to see what happens in the UL if you call twice the "solve" function.

also check if you advance in time and you apply an increment if it is still ok